### PR TITLE
feature/67-give-the-storybook-a-custom-factiverse-logo

### LINKED ISSUE #67: Give the storybook a custom Factiverse Logo
**Is your feature request related to a problem? Please describe.**
Update the Storybook theme to be more  life 'Factiverse'

**Describe the solution you'd like**
- Add our logo
- Color the text and look and feel
- Change the font

**Additional context**
Use the storybook theming plugin: https://storybook.js.org/docs/react/configure/theming

### LINKED ISSUE #68: The Factiverse images are not bundled into the component library
## Bug Report

**Current behavior**
A clear and concise description of the behaviour.

**To Reproduce**

1. When running the build script in the storybook, `npm run build`
2. Go to the newly created `/build` folder.
3. See the sub folders.
4. See that the `/build/img` folder is not present

**Expected behavior**
There should be a `/build/img` folder present in the bundled folder.

**Possible Solution**

Add the src and dest fields to the `rollup.config.js` file in the root dir. See screenshot for clarity.

You will know this works if the `build/img` folder is present and that the component renders with the logo.

**Screenshots**
This file needs an update here:

Screenshot 2022-09-20 at 12 13 54

### PR DESCRIPTION
**Pull request by DomyS**
Added logo and img folder as described in #68 

Closes #68 
Closes #67 

## Screenshots (before and after the change):
added this logo 
Untitled (1)

### CODE REVIEW

File: .storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { create } from '@storybook/theming';
+import Logo from './Logo.png';
```
Comment (withchristopher) [inline .storybook/theme.js:2]: Should this not be in the img folder but not copied over to the build folder.

File: .storybook/theme.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+export default create({
+  base: 'light',
+  brandTitle: 'Factiverse Design System',
+  brandUrl: 'http://factiverse.no/',
```
Comment (withchristopher) [inline .storybook/theme.js:7]: This should be the factiverse storybook URL 

`https://factiverse.github.io/storybook-design-system/?path=/story/base-components-autocomplete--with-data`

**Review by withchristopher — CHANGES_REQUESTED** [review]
Nice work! Just a few small things. I also did not test exporting the component library locally yet.

File: .storybook/theme.js (hunk 1 above)
Comment (DomyS) [reply .storybook/theme.js]: Fixed

File: .storybook/theme.js (hunk 2 above)
Comment (DomyS) [reply .storybook/theme.js]: Fixed

**Review by withchristopher — APPROVED** [review]
LGTM

**Review by withchristopher — APPROVED** [review]

**Comment by DomyS** [comment]
Can somebody check Storybook Publish and UI Tests? can not merge if these are not approved

**Comment by withchristopher** [comment]
I have approved it but still not reflecting. Maybe run again?

**Comment by tobiastykvart** [comment]
Chromatic was down yesterday when I tried to approve it, maybe something got messed up. Try doing a new commit to make the checks run again.

**Pull request merged**